deploy/flatten/che-workspace-crd.package.yaml

```diff
@@ -1,4 +1,4 @@
 packageName: che-workspace-crd
 channels:
 - name: alpha
-  currentCSV: che-workspace-crd-operator.v0.0.1
```

Comment:
Don't you also rename the file ?

Reply:
The file is renamed to `che-workspace-operator.package.yaml`